ezyrb/ann.py

```diff
@@ -157,3 +156,39 @@ def predict(self, new_point):
         new_point = self._convert_numpy_to_torch(np.array(new_point))
         y_new = self.model(new_point)
         return self._convert_torch_to_numpy(y_new)
+    
+    def predict_tensor(self, new_point):
+        
+        return self.model(new_point)
```

Comment:
Such a method is just a type specification of the `predict` method. Please avoid that, and instead check the type inside `predict` and use conversion only if the input is NumPy